cli/installer/install-azd.ps1

```diff
@@ -61,6 +61,51 @@ param(
     [int] $DownloadTimeoutSeconds = 120
 )
 
+
+# Windows specific:
+# This functions sends a WM_SETTINGCHANGE message which causes new processes to
+# pick up the updates to environment variables. Not calling this funciton after
+# updating environment variables means that new processes will use an older view
+# of the environment variables.
+function broadcastSettingChange {
+    $SEND_MESSAGE_TIMEOUT_DEFINITION = @"
+    [DllImport("user32.dll")]public static extern IntPtr SendMessageTimeout(
+        IntPtr hWnd,
+        uint Msg,
+        UIntPtr wParam,
```

Comment:
Nit: you're always passing Zero so you don't need to use `[UIntPtr]` here. Technically, signed/unsigned only matters if it would overflow. Same for any `uint`s below. You have 1 specific case, so could just keep it simple.